docs/schema/modification_guid_name-element-workflow.md

```diff
@@ -68,7 +69,7 @@ None
 
 The modification name is displayed as the link text for the modification link on the workflow status page.
 
-To specify a modification form, you must first set the <span sdata="cer" target="P:Microsoft.SharePoint.WorkflowActions.EnableWorkflowModification.ModificationId"><span class="nolink">ModificationId</span></span> property of the <span sdata="cer" target="T:Microsoft.SharePoint.WorkflowActions.EnableWorkflowModification"><span class="nolink">EnableWorkflowModification</span></span> activity to the GUID you generated for your modification form. You can have multiple modifications available in your workflow; you must assign each modification a unique identifier.
+To specify a modification form, you must first set the Microsoft.SharePoint.WorkflowActions.EnableWorkflowModification.ModificationId">ModificationId property of the T:Microsoft.SharePoint.WorkflowActions.EnableWorkflowModification activity to the GUID you generated for your modification form. You can have multiple modifications available in your workflow; you must assign each modification a unique identifier.
```

Comment:
You need to get rid of all the span tags here.  Line 72 should look like this:

To specify a modification form, you must first set the ModificationId property of the EnableWorkflowModification activity to the GUID you generated for your modification form. You can have multiple modifications available in your workflow; you must assign each modification a unique identifier.